api/opds_for_distributors.py

```diff
@@ -306,12 +306,7 @@ def update_work_for_edition(self, *args, **kwargs):
         pool, work = super().update_work_for_edition(
             *args, is_open_access=False, **kwargs
         )
-        pool.update_availability(
-            new_licenses_owned=1,
-            new_licenses_available=1,
-            new_licenses_reserved=0,
-            new_patrons_in_hold_queue=0,
-        )
+        pool.unlimited_access = True
```

Comment:
The docstring for this method is misleading. OPDS for Distributors books can be either Open Access or unlimited access and should be assigned appropriately, depending on the acquisition link in the feed:
- `rel="http://opds-spec.org/acquisition"` for unlimited access and
- `rel="http://opds-spec.org/acquisition/open-access"` for OA

The current `OPDSForDistributorsImporter` appears to respect this, since I can see at least one production collection that contains both. I'm not entirely sure where we handle the OA part, though.

Reply:
I'm not entirely sure what to do about this. I haven't changed any behaviour with regards to whether or not something is assigned open access status, so it should continue to do whatever it did before with respect to the link relations.